docs/03-keybind-overview.md

```diff
@@ -21,6 +21,7 @@ by pressing `<backspace>` in the which-key main menu (first popup after pressing
 | `<leader>`          | [whichkey](https://github.com/folke/which-key.nvim) (keybinds popup (shows up after 1s)) | normal |
 | `<leader>e`         | [nvimtree](https://github.com/nvim-tree/nvim-tree.lua) (side file explorer)              | normal |
 | `<leader>f`         | [telescope](https://github.com/nvim-telescope/telescope.nvim) (find files)               | normal |
+| `<leader>st`        | [telescope](https://github.com/nvim-telescope/telescope.nvim) (grep)                     | normal |
```

Comment:
```suggestion
| `<leader>f`  `<leader>s`(menu) | [telescope](https://github.com/nvim-telescope/telescope.nvim) (find files, grep text, and more)               | normal |

maybe something more like this? I don't want this list to become a copy of whichkey
```

Reply:
> maybe something more like this? I don't want this list to become a copy of whichkey

I think that's a fair balance @LostNeophyte; gives newbies access to the basics to get started without making the documentation too verbose.